actors/miner/src/types.rs

```diff
@@ -529,3 +529,14 @@ pub struct GetVestingFundsReturn {
 }
 
 impl Cbor for GetVestingFundsReturn {}
+
+#[derive(Debug, Default, PartialEq, Eq, Clone, Serialize_tuple, Deserialize_tuple)]
```

Comment:
Question whether to make this transparent. Unfortunately ChangePeerID already has one parameter but not transparent.

Reply:
Yeah, that was why I went with this being not-transparent. I would've preferred to do so transparently, but decided to prioritize consistency.